app/presenters/topic_presenter.rb

```diff
@@ -33,7 +35,7 @@ def phase
   end
 
   def visually_collapsed?
-    @visually_collapsed
+    visually_collapsed
   end
```

Comment:
```suggestion
alias_method :visually_collapsed?, :visually_collapsed
```

This might be an alternative to having the method, but I don't really mind. It's sad we can't define the variable with the question mark.

Reply:
Ohh, didn't think of using an alias - I think it makes more sense than just a simple method so thanks for the suggestion 👍  

Yeah it is a little, especially when the value is a boolean. I suppose Rails developers normally get round this explicitly by using a method like `present?` which isn't as nice.